src/components/book-preview.ts

```diff
@@ -1,38 +1,53 @@
 import { Component, Input } from '@angular/core';
 
 import { Book } from '../models';
+import { MD_CARD_DIRECTIVES } from '@angular2-material/card';
+import { MD_LIST_DIRECTIVES } from '@angular2-material/list';
 
 
 export type BookInput = Book;
 
 @Component({
   selector: 'book-preview',
+  directives: [
+    ...MD_CARD_DIRECTIVES,
+    ...MD_LIST_DIRECTIVES
+  ],
   template: `
-    <a [linkTo]=" '/book/' + id ">
-      <h3>{{ title }}</h3>
+    <a [linkTo]=" '/book/' + id">
+      <md-card>
+        <md-card-title-group>
+          <md-card-title>{{ title }}</md-card-title>
+          <md-card-subtitle *ngIf="subtitle">{{ subtitle }}</md-card-subtitle>
+          <img md-card-sm-image *ngIf="thumbnail" [src]="thumbnail"/>
+        </md-card-title-group>
+        <md-card-content>
+          <p *ngIf="description">{{ description | slice: 0:250 }}{{ description.length > 20 ? '...' : '' }}</p>
```

Comment:
This seems like a good candidate for a pipe, though there's a _small_ issue. The description could contain HTML fragments. Not sure what the best solution is for this just yet.